src/image/p5.Image.js

```diff
@@ -220,6 +220,40 @@ p5.Image = class {
     this.pixels = [];
   }
 
+  /**
+   * Set the pixel density of the image.
+   *
+   * @method setPixelDensity
+   * @param {Number} density - The pixel density to set.
+   * @example
+   * <div><code>
+   * let img = new p5.Image(100, 100);
+   * img.setPixelDensity(2);
+   * </code></div>
+   */
+  setPixelDensity(density) {
```

Comment:
Do you think we could match the API of main canvases and `p5.Graphics` where `pixelDensity()` returns the current density, and `pixelDensity(newDensity)` sets it?

Reply:
Let me look into what you are saying and get back to you. Thanks for the feedback.